SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -52,6 +52,11 @@ FString FSpatialGDKEditorModule::GetSpatialOSLocalDeploymentIP() const
 	return GetDefault<USpatialGDKEditorSettings>()->ExposedRuntimeIP;
 }
 
+bool FSpatialGDKEditorModule::ShouldStartPIEClientsWithLocalLaunchOnDevice() const
```

Comment:
`LocalLaunch` here being spatial yeah?
Might get confusing in the future with `Launch` meaning `launch on device` with unreal. 
Perhaps we should say `LocalDeployment` ?

Reply:
It's a combination of the two in this case, `Launch on device` running with `Local deployment` flow selected for Spatial.